### PR TITLE
Minor prerelease updates

### PR DESCRIPTION
**Pull request by dingo-d**
I've updated the changelog and replaced old links of WPCS github repository with the new one.

Once this is merged I can tag and release 0.2.0 version 🎉

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,54 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
...
+  Or you can check the in depth documentation about the [template hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy).
+- New `WPThemeReview.Privacy.ShortenedURLs` sniff: detects the usage of shortened URLs. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#privacy).
+- New `WPThemeReview.CoreFunctionality.PostsPerPage` sniff: adds a warning if `-1` is used in `posts_per_page` setting while querying posts, due to detrimental effects it has on the query speed.
+- The `WordPress.PHP.IniSet` rule was added to the ruleset about setting ini configuration during runtime.
+- The `WordPress.WP.DeprecatedParameterValues` rule was added to the ruleset about usage of deprecated parameter values in WP functions. The sniff will provide alternative based on the parameter passed.
```
Comment (jrfnl) [inline CHANGELOG.md]:
```suggestion
- The `WordPress.WP.DeprecatedParameterValues` rule was added to the ruleset about usage of deprecated parameter values in WP functions. The sniff will suggest an alternative based on the parameter passed.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,54 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
...
+    The default property value can be overruled and/or extended from a custom ruleset.
+    * Similar to the WPCS FileNameSniff, this sniff does not currently allow for mimetype sublevel only theme template file names, such as `plain.php`.
+
+### Changed
+
+- Travis updates:
+  - Run the code style related and ruleset checks in separate stages.
+  - Removed `sudo: false` setting.
+  - Test agains high/low WPCS versions due to sniffs that are extending the WPCS native sniffs.
+- Update forbidden functionality with cron functions in the `WPThemeReview.PluginTerritory.ForbiddenFunctions` sniff.
```
Comment (jrfnl) [inline CHANGELOG.md]: While this is a change to the sniff, it results in new checks being added to TRTCS, so IMO, this entry should be in the `Added` section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,54 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
...
+- Added XSD schema tags and validated the ruleset against schema (PHPCS 3.2+/3.3.2+).
+- Update forbidden functionality with (un)register editor blocks functions in the `WPThemeReview.PluginTerritory.ForbiddenFunctions` sniff.
```
Comment (jrfnl) [inline CHANGELOG.md]: Same remark as for the cron functions.
These two entries could possibly be combined into one.

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,54 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
...
+- Updated the WPCS to 2.1.0 version.
+- Updated the suggested installer version.
+- Composer tweaks: improve readability of script section
+- Ruleset tweaks: limit PHPCompatibility to PHP files
+- Update the WPCS namespace to `WordPressCS`
```
Comment (jrfnl) [inline CHANGELOG.md]:
```suggestion
- Updated the WPThemeReview codebase, where relevant, for compatibility with WPCS 2.0+.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,54 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
...
+
+### Fixed
+
+- Removal of HTML from error message about adding menu pages in `WPThemeReview.PluginTerritory.NoAddAdminPages` sniff.
+- Minor grammar changed in ruleset.
```
Comment (jrfnl) [inline CHANGELOG.md]:
```suggestion
- Minor grammar changes in the ruleset.
```

**Comment by dingo-d** [comment]
Thanks for the review, I've commited the changes.

> It contains a lot of details, some of which is not really interesting or even relevant to the users of the repo. So my question is: who is your target audience for the changelog ?

The target audience should be the users of the ruleset, so I've removed some extra information. Most of the sniffs have extra information in their docblocks, and I plan on adding the documentation like in the WPCS so that extra information can be added for every sniff.

> What is the logic for the order of the entries in each section?

I went to the commit list and just went commit by commit from the last commit to the first one. So the order logic is chronological.

I've also closed the issues or moved them to the next release, so we're good there :+1:

**Pull request merged**